index.html

```diff
@@ -1,176 +1,780 @@
-<!doctype html>
 
+<!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="utf-8">
+  <head>
 
-    <title>Quanteco</title>
-    <meta name="description" content="Quanteco.org">
-    <meta name="author" content="Quanteco">
-    <meta name="referrer" content="no-referrer">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <!-- material design lite -->
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.green-blue.min.css" />
-    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
-
-    <!-- fonts -->
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
 
-    <link rel="stylesheet" href="style.css">
```

Comment:
Do we no longer have our own main stylesheet? All styles are from the template then? I'm confused

Reply:
Everything from the template